zeppelin/converter.py

```diff
@@ -4,17 +4,15 @@
 import os
 from datetime import datetime
 
-"""
-ZeppelinConverter is a utility to convert Zeppelin raw json into Markdown
-
-"""
-
 
 class ZeppelinConverter:
-
+    """
+    ZeppelinConverter is a utility to convert Zeppelin raw json into Markdown
+    """
     MD_EXT = '.md'
 
     def __init__(self, ifn, ofn, dire, user, date_created, date_updated):
+        """ Constructor """
```

Comment:
There's no need to say that this is the constructor, it's name is already saying that.